wtf.py

```diff
@@ -1,10 +1,11 @@
 #!/usr/bin/env python
```

Comment:
I would change `python` to `python3` here. See https://github.com/dlenski/wtf/issues/26.